tests/webtest_tests.py

```diff
@@ -510,6 +510,21 @@ def test_merged_user_has_alert_message_on_profile(self):
         res = self.app.get('/profile/{0}/'.format(self.dupe._primary_key)).maybe_follow()
         assert_in('This account has been merged', res)
 
+    def test_merged_user_with_two_account_on_same_project_with_different_visibility_and_permissions(self):
```

Comment:
This isn't really a webtest. This is testing model logic, so should go in `test_models.py`.